brands/amenity/bank.json

```diff
@@ -1278,24 +1278,6 @@
       "name:es": "Banco Pichincha"
     }
   },
-  "amenity/bank|Banco Popular": {
-    "countryCodes": ["us", "vi"],
-    "matchNames": ["popular"],
-    "tags": {
-      "amenity": "bank",
-      "brand": "Banco Popular",
-      "brand:en": "Popular Bank",
-      "brand:es": "Banco Popular",
-      "brand:wikidata": "Q537259",
-      "brand:wikipedia": "es:Banco Popular Español",
```

Comment:
Banco Popular Español was a bank in Spain. It did have a sister company in the U.S., but not under that name.